### DISCUSSION TITLE
[Feature] Search by file path

### DISCUSSION THREAD

**Discussion by vdeconinck** [immich-app/immich#14210] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

My external library contains around 150k pictures, grouped by folders named according to the event (e.g. 10-year anniversary) or topic (e.g. car crash damages) but Immich doesn't seem to index the path (seems I can only search on the filename in the search options).

Is there something I'm missing ?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#6519
